src/queries/queries.js

```diff
@@ -49,3 +65,28 @@ export const buildGeoJSONQueryNoLimit = () => ({
     fetchPolicy: CozyClient.fetchPolicies.olderThan(older30s)
   }
 })
+
+export const buildGeoJSONQueryNoLimitByDate = date => {
```

Comment:
nit: Tant que nous utiliserons `UNSAFE_noLimit`, je serais d'avis le mettre en dernier dans le nom.
(Comme les 2 précédentes)
```suggestion
export const buildGeoJSONQueryByDateNoLimit = date => {
```